modules/microshift-provisioning-ostree.adoc

```diff
@@ -3,38 +3,23 @@
 // microshift/microshift-embed-into-rpm-ostree.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="provisioning-a-machine_{context}"]
-= Provisioning a machine for {microshift-short}
+[id="miroshift-provisioning-a-virtual-machine_{context}"]
+= Provisioning a {op-system-ostree} virtual machine for {microshift-short} using a Kickstart file
```

Comment:
I'm not sure about changes in this section. It does not describe provisioning, but creating a kickstart and embedding it into the ISO.

Reply:
Hmm, looks like it has said "provisioning" from the very beginning, https://github.com/openshift/openshift-docs/pull/85240/files#diff-9bfa2bcc1f8b71b37199051d9b95d2eca41d1e586472a2a4d6606a9118ad7bc6L7. We say right below "provision using the RHEL for Edge documentation," so agree, this is clearly the wrong heading.